src/03-control-flow.js

```diff
@@ -128,7 +159,25 @@ function aroundTheWorldGreetingWithSwitch(language) {}
  *  calculateLetterGrade(99);
  *  //> "A"
  */
-function calculateLetterGrade(grade) {}
+function calculateLetterGrade(grade) {
```

Comment:
Remember we don't need a `break` statement if we have a `return` in the switch.  The return will stop the execution of the case and return the value we specify.